protocol/protocol_test.go

```diff
@@ -370,6 +371,11 @@ func TestIETFTags(t *testing.T) {
 		got  uint32
 		want uint32
 	}{
+		{
+			name: "SRV",
+			got:  tagSRV,
+			want: 0x00565253,
```

Comment:
Reviewer note: The bytes in Table 2 are incorrect: https://github.com/wbl/roughtime-draft/pull/10